src/instructlab/configuration.py

```diff
@@ -26,6 +26,8 @@
 import httpx
 import yaml
 
+LLAMA_CPP = "llama-cpp"
+VLLM = "vllm"
```

Comment:
Why did these two move out of "backend"?

Reply:
as with most other constants that live in the config.yaml, IMO these should live in configuration.py. most places where we import these we import config.py anyway for other constants.

Just trying to align with the principles we have been following elsewhere

Reply:
Ok thanks!